.ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -0,0 +1,149 @@
+
+worker_nodes=$(oc get nodes |  awk '{if ($3 == "worker") { print $1 } }')
+num_nodes=$(echo $worker_nodes | wc -w)
+[ $num_nodes -ne 0 ] || \
+	die "No worker nodes detected. Something is wrong with the cluster"
+
+info "Set installing state annotation and label on all worker nodes"
+for node in $worker_nodes; do
+	info "Deploy kata in worker node $node"
+	oc annotate --overwrite nodes $node \
+		kata-install-daemon.v1.openshift.com/state=installing
+	oc label --overwrite node $node kata-deploy=true
+done
+
+info "Applying the kata installer daemonset"
+if [ -z "$KATA_INSTALLER_IMG" ]; then
+	# The yaml file uses $IMAGE_FORMAT which gives the
+	# registry/image_stream:image format
+	export KATA_INSTALLER_IMG=$(echo $IMAGE_FORMAT"kata-installer" \
+		| envsubst)
+fi
+envsubst < ${deployments_dir}/daemonset_kata-installer.yaml.in | oc apply -f -
+
+oc get pods
+oc get ds
+ds_pods=($(oc get pods | awk '{if ($1 ~ "kata-deploy") { print $1 } }'))
+cnt=5
+while [[ ${#ds_pods[@]} -gt 0 && $cnt -ne 0 ]]; do
+	sleep 120
+	for i in ${!ds_pods[@]}; do
+		info "Check daemonset ${ds_pods[i]} is running"
+		rc=$(oc exec ${ds_pods[i]} -- cat /tmp/kata_install_status 2> \
+			/dev/null)
+		if [ -n "$rc" ]; then
+			info "Finished with status: $rc"
+			oc describe pods ${ds_pods[i]}
+			oc logs ${ds_pods[i]}
+			unset ds_pods[i]
+		else
+			info "Running"
+		fi
+	done
+	cnt=$((cnt-1))
+done
+
+if [ $cnt -eq 0 ]; then
+	for p in ${ds_pods[@]}; do
+		info "daemonset $p did not finish"
+		oc describe pods $p
+		oc logs $p
+	done
+	die "Kata Containers seems not installed on some nodes"
+fi
+
+# Finally remove the installer daemonset
+info "Deleting the kata installer daemonset"
+oc delete -f ${deployments_dir}/daemonset_kata-installer.yaml.in
+
+# Apply the CRI-O configuration
+info "Configuring kata runtime for CRI-O"
+if [ -z "$KATA_CRIO_CONF_BASE64" ]; then
+	export KATA_CRIO_CONF_BASE64=$(echo \
+		$(cat $configs_dir/crio_kata.conf|base64) | sed -e 's/\s//g')
+fi
+envsubst < ${deployments_dir}/machineconfig_kata_runtime.yaml.in | oc apply -f -
+if [ $(oc get machineconfigs|grep 50-kata|wc -l) != "0" ]; then
+	echo "Machineconfig was found"
+else
+	echo "Waiting for machineconfig object to show up"
+fi
+
+# The machineconfig which installs the kata drop-in in CRI-O will trigger a
+# worker reboot.
+wait_for_reboot 1800
+
+# Add a runtime class for kata
+info "Adding the kata runtime class"
+oc apply -f ${deployments_dir}/runtimeclass_kata.yaml
+oc get runtimeclass/kata || die "kata runtime class not found"
+
+# Set SELinux to permissive mode
+info "Configuring SELinux"
+if [ -z "$SELINUX_CONF_BASE64" ]; then
+	export SELINUX_CONF_BASE64=$(echo \
+		$(cat $configs_dir/selinux.conf|base64) | sed -e 's/\s//g')
+fi
+envsubst < ${deployments_dir}/machineconfig_selinux.yaml.in | oc apply -f -
+oc get machineconfig/51-kata-selinux || die "SELinux machineconfig not found"
+# The new SELinux configuration will trigger another reboot.
+wait_for_reboot 1800
```

Comment:
Nit: duplicated timeout value - maybe define a variable at the top of the file to avoid duplication?

Reply:
Done. Now the `wait_for_reboot` has 1800 as default timeout.